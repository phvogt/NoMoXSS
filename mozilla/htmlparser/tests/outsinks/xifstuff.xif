<?xml version="1.0"?>
<!DOCTYPE xif>
<encode selection="0"/>
<section>
<section_head>
<document_info charset="UTF-8"/>
</section_head>
<section_body>
<container isa="html">
<content>
</content>
<container isa="head">
<content>
</content>
<comment><content>
  -- The contents of this file are subject to the Netscape Public
  -- License Version 1.1 (the "License"); you may not use this file
  -- except in compliance with the License. You may obtain a copy of
  -- the License at http://www.mozilla.org/NPL/
  --  
  -- Software distributed under the License is distributed on an "AS
  -- IS" basis, WITHOUT WARRANTY OF ANY KIND, either express or
  -- implied. See the License for the specific language governing
  -- rights and limitations under the License.
  --  
  -- The Original Code is Mozilla Communicator client code, released
  -- March 31, 1998.
  -- 
  -- The Initial Developer of the Original Code is Netscape
  -- Communications Corporation. Portions created by Netscape are
  -- Copyright (C) 1998-1999 Netscape Communications Corporation. All
  -- Rights Reserved.
  -- 
  -- Contributor(s): 
  </content>
</comment>
<content>

</content>
<container isa="title">
<content>XIF Test Page</content>
</container><!--title-->
<content>

</content>
<leaf isa="meta">
<attr name="http-equiv" value="Content-Type"/>
<attr name="content" value="text/html; charset=utf-8"/>
</leaf><!--meta-->

</container><!--head-->

<content>

</content>

<container isa="body">
<content>
</content>
<container isa="h2">
<content>Here's the deal...</content>
</container><!--h2-->
<content>

</content>

<container isa="p">
<content>This is a good place to add in </content>
<container isa="b">
<content>html</content>
</container><!--b-->
<content> to aid in testing features 
under development.
</content>

<entity value="lt"/><content>a href="http://www.mozilla.org"</content>
<entity value="gt"/></content>

<content>
It's also a great place to not use latin.
</content>

<comment><content> This is a comment;
     Here is more of the comment.
 </content>
</comment>
<content>

</content>

</container><!--p-->
</container><!--body-->
</container><!--html-->
</section_body>
</section>
