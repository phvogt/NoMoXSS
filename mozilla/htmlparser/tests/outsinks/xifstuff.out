<html>
<head>
<!--
  -- The contents of this file are subject to the Netscape Public
  -- License Version 1.1 (the "License"); you may not use this file
  -- except in compliance with the License. You may obtain a copy of
  -- the License at http://www.mozilla.org/NPL/
  --  
  -- Software distributed under the License is distributed on an "AS
  -- IS" basis, WITHOUT WARRANTY OF ANY KIND, either express or
  -- implied. See the License for the specific language governing
  -- rights and limitations under the License.
  --  
  -- The Original Code is Mozilla Communicator client code, released
  -- March 31, 1998.
  -- 
  -- The Initial Developer of the Original Code is Netscape
  -- Communications Corporation. Portions created by Netscape are
  -- Copyright (C) 1998-1999 Netscape Communications Corporation. All
  -- Rights Reserved.
  -- 
  -- Contributor(s): 
  -->

<title>XIF Test Page</title>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8"></head>

<body>
<h2>Here's the deal...</h2>

<p>This is a good place to add in <b>html</b> to aid in testing features 
under development.
&lt;a href="http://www.mozilla.org"&gt;
It's also a great place to not use latin.
<!-- This is a comment;
     Here is more of the comment.
 -->

</p></body>
</html>
